Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```diff
@@ -152,6 +152,13 @@ configuration:
   section: Connect
   advanced: true
   required: false
+- display: Log type max number of job pulling attempts
+  name: fetch_job_polling_max_num_attempts
+  additionalinfo: "The maximum number of attempts to try and pull results for each log type from a job that was created by its query. Each attempt takes around 1 second. Increasing this value is useful in case there are many logs to pull from a given log type.\nNote: When increasing this number, in case fetching more than 4 logs types together, it is recommended to split different log types for different integration instances"
+  type: 0
+  defaultvalue: 10
+  advanced: true
+  section: Collect
```

Comment:
This parameter doesn't seem to be in-use anywhere in the added Python code?

Reply:
This is related to updates in the integration since I opened this PR. This should get resolved after I pull & merge from master.